recipe/yum_requirements.txt

```diff
@@ -1 +1,2 @@
+mesa-libGL
```

Comment:
Build was [failing]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=435995&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642 ) (also [log attached]( https://github.com/conda-forge/pyglet-feedstock/files/7813967/linux_64_python3.10.____cpython_435995_47.txt )) due to GLX support not being present in the X Server. Hopefully this fills that gap